### PR TITLE
fix: addPolicy slice affect

### PR DESCRIPTION
**Pull request by maxiloEmmmm**
Fix: https://github.com/casbin/casbin/issues/663

### CODE REVIEW

File: management_api.go [hunk 1]
```diff
@@ -134,9 +134,14 @@ func (e *Enforcer) AddPolicies(rules [][]string) (bool, error) {
 // Otherwise the function returns true by adding the new rule.
 func (e *Enforcer) AddNamedPolicy(ptype string, params ...interface{}) (bool, error) {
 	if strSlice, ok := params[0].([]string); len(params) == 1 && ok {
-		return e.addPolicy("p", ptype, strSlice)
+		set := make([]interface{}, 0, len(strSlice))
```
Comment (hsluoyz) [inline management_api.go:137]: Why is the code logic changed?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@maxiloEmmmm plz fix all checks.

File: management_api.go (hunk 1 above)
Comment (maxiloEmmmm) [reply management_api.go:137]: see test file and https://github.com/casbin/casbin/issues/663

**Comment by hsluoyz** [comment]
@maxiloEmmmm DCO and Semantic PR

**Comment by maxiloEmmmm** [comment]
> @maxiloEmmmm DCO and Semantic PR

How to fix Semantic PR?
Edit history commit message or new semantic commit message?
always get ×

**Pull request closed without merging**